### PR TITLE
:lipstick: Location message  block

### PR DESCRIPTION
**Pull request by Dancunrandy**
# Description

Redesigned location message block

Fixes #829 
## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# Screenshot (optional)
Screenshot from 2023-11-08 15-09-26
Screenshot from 2023-11-08 15-09-58

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

**Test Configuration**: (optional)
* Firmware version:
* Hardware:
* Toolchain:
* SDK:

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by Dancunrandy** [comment]
@LemmyMwaura , this is the one i closed by mistake and had to create new PR

File: libs/features/convs-mgr/stories/blocks/library/main/src/lib/utils/transformers/location-block.jsplumb.ts [hunk 1]
```diff
@@ -22,9 +22,16 @@ export function _LocationBlockDecoratePlumb(block: LocationMessageBlock, comp: C
...
   });
-
+  // jsPlumb.addEndpoint(comp.location.nativeElement, {
+  //   // Whether the anchor is target (Other Block -> This Block)
+  //   target: true,
+  //   cssClass:"block_endpoint",
+  //   endpoint: "Dot",
+  //   anchor: [1, 0.9 ,0, 1],
+  //   maxConnections: -1
+  // });
```
Comment (LemmyMwaura) [inline libs/features/convs-mgr/stories/blocks/library/main/src/lib/utils/transformers/location-block.jsplumb.ts:35]: remove dead code

File: libs/features/convs-mgr/stories/blocks/library/location-message-block/src/lib/components/location-block/location-block.component.html [hunk 2]
```diff
@@ -15,7 +15,7 @@
 			<map-marker [position]="markerPositions" [options]="markerOptions"></map-marker>
 		</google-map>
 
-		<app-default-option-field [jsPlumb]="jsPlumb" [blockFormGroup]="locationMessageForm">
+		<app-default-option-field [jsPlumb]="jsPlumb" className="app-default-option-field" [blockFormGroup]="locationMessageForm">
```
Comment (LemmyMwaura) [inline libs/features/convs-mgr/stories/blocks/library/location-message-block/src/lib/components/location-block/location-block.component.html:18]: No need for this className

**Review by LemmyMwaura — CHANGES_REQUESTED** [review]

**Comment by Dancunrandy** [comment]
@LemmyMwaura have updated the code

File: libs/features/convs-mgr/stories/blocks/library/main/src/lib/utils/transformers/location-block.jsplumb.ts [hunk 3]
```diff
@@ -22,9 +22,8 @@ export function _LocationBlockDecoratePlumb(block: LocationMessageBlock, comp: C
     target: true,
     cssClass:"block_endpoint",
     endpoint: "Dot",
-    anchor: "Left",
+    anchor: [0, 0.1,0,1],
```
Comment (LemmyMwaura) [inline libs/features/convs-mgr/stories/blocks/library/main/src/lib/utils/transformers/location-block.jsplumb.ts:25]: Let's leave this as "Left" as it was for now

**Review by LemmyMwaura — COMMENTED** [review]
Small change left, Ping me when you are done.

**Comment by Dancunrandy** [comment]
@LemmyMwaura , have restored anchor position

**Comment by Dancunrandy** [comment]
@LemmyMwaura PR updated.

**Pull request closed without merging**